### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by amarphule**

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]
Code looks good. Just you can use rems at some places instead of px.

**Review by narharikale — APPROVED** [review]
seems good to me 👍

**Comment by amarphule** [comment]
> Code looks good. Just you can use rems at some places instead of px.

Thank you for the suggestion, it makes sense. Made changes.

**Comment by amarphule** [comment]
> seems good to me +1

Thank you!

**Comment by amarphule** [comment]
Made suggested changes.

File: index.html [hunk 1]
```diff
@@ -55,6 +55,42 @@ <h1>
...
+            img tag src attribute contain image path.
+          </p>
+          <div class="inline">
+            <div class="avatar-sm">
+              <img src="./assets/avatar.png" alt="avatar-small" />
+            </div>
+            <div class="avatar-md">
+              <img src="./assets/avatar.png" alt="avatar-medium" />
+            </div>
+            <div class="avatar-l">
```
Comment (SanjayJatti) [inline index.html:72]: for large size avatar, you can add "avatar-lg" instead of "avatar-l". I will be more consistent with sm and md.

**Comment by SanjayJatti** [comment]
All good nice to me.

**Pull request merged**